src/main/java/com/backendoori/ootw/common/validation/ImageValidator.java

```diff
@@ -0,0 +1,25 @@
+package com.backendoori.ootw.common.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.springframework.web.multipart.MultipartFile;
+
+public class ImageValidator implements ConstraintValidator<ImageValid, MultipartFile> {
+
+    @Override
+    public boolean isValid(MultipartFile img, ConstraintValidatorContext context) {
+        if(img == null || img.isEmpty()){
+            return false;
+        }
+        if(img.getSize() > 10_000_000){
+            return false;
+        }
+        String contentType = img.getContentType();
+        if(!contentType.startsWith("image")){
+            return false;
+        }
+
+        return true;
```

Comment:
가독성을 구분하는 습관이 정말 좋네요
저는 개인적으로 boolean 반환 메서드의 경우 아래와 같이 많이 쓰긴 합니다

```suggestion
        String contentType = img.getContentType();
        
        return contentType.startsWith("image");
```